2-mandatory/3-financial-times.js

```diff
@@ -5,33 +5,75 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    let newArray = [];
+    for (let i = 0; i < allArticleTitles.length; i++) {
+        if (allArticleTitles[i].length <= 65) {
+            newArray.push(allArticleTitles[i]);
+        }
+    }
+    return newArray;
 }
 
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
-    (you can assume words will always be seperated by a space)
+    (you can assume words will always be separated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
-}
+  let newString=''
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (
+      newString.length < 1 ||
+      newString.split(" ").length > allArticleTitles[i].split(" ").length
+    ) {
+      newString = allArticleTitles[i];
+    }
+
+  }
+
+   return newString;
+} 
+
```

Comment:
Another way of doing this is to initialise newString with the first entry, and then you can just check each subsequent entry with only one condition